vendor/bat-native-ledger/src/bat/ledger/internal/wallet/balance.cc

```diff
@@ -101,6 +103,11 @@ void Balance::OnWalletProperties(
 void Balance::GetUnBlindedTokens(
     ledger::BalancePtr balance,
     ledger::FetchBalanceCallback callback) {
+  if (!balance) {
```

Comment:
If `balance` was a `nullptr` before this defensive code was added, could it have caused any side effects other than a crash?